util.go

```diff
@@ -34,21 +37,23 @@ func extractTTL(ipPkt []byte) (uint8, error) {
 		return 0, err
 	}
 
-	return uint8(ipPkt[8]), nil
+	return uint16(ipPkt[4])<<8 | uint16(ipPkt[5]), nil
```

Comment:
Would this be easier to read as

```go
// IP Identification field is in bytes 4-5 of the header.
return binary.BigEndian.Uint16(ipPkt[4:])
```
(from encoding/binary in the standard library)?

Reply:
Agreed. Fixed in aac5e581c3ef92c4f3938d634a8605e8533255d8.